BuildingSync.xsd

```diff
@@ -14308,6 +14308,8 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
 								</xs:annotation>
 								<xs:simpleType>
 									<xs:restriction base="xs:string">
+										<xs:enumeration value="Tenant Area"/>
+										<xs:enumeration value="Common Area"/>
```

Comment:
Could we change these two to "Tenant" and "Common", I think "Area" is redundant.